### DISCUSSION TITLE
Mouse selection doesn't seem to work.

### DISCUSSION THREAD

**Discussion by iliasaz** [krzyzanowskim/STTextView#20] [Questions]
Hi,
It appears that the simplest selection mode, mouseDown-and-drag, doesn't work. 

Other selection modes works fine:
Double-click selects the word clicked
Triple click selects the line clicked
Shift-click selects the range
keyboard Shift-combinations all work fine.

Am I missing something, or is it just not supported?
Thanks!

**Comment by krzyzanowskim** [comment]
I'm not aware it not being supported. Recently there was a change for dragging text with a drag gestuee. I wonder if that's something that you may feel as not working. Can you try 0.7.0?

**Reply by iliasaz** [reply]
Aha, 0.7.0 works fine!

**Reply by krzyzanowskim** [reply]
I see what you mean by "not working". I improved it on main branch now.

**Reply by krzyzanowskim** [reply]
0.7.2 should have it fixed/improved. Please let me know how it looks on your side

**Reply by iliasaz** [reply]
Nice! Thank you!
